lib/identity_config.rb

```diff
@@ -358,6 +358,11 @@ def self.build_store(config_map)
     config.add(:usps_ipp_username, type: :string)
     config.add(:usps_ipp_request_timeout, type: :integer)
     config.add(:usps_upload_enabled, type: :boolean)
+    config.add(:get_usps_proofing_results_job_cron, type: :string, allow_nil: true)
+    config.add(
+      :get_usps_proofing_results_job_reprocess_delay_minutes, type: :string,
```

Comment:
Could we make this `type: :integer` so that we don't have to `.to_f` in the job itself?

Reply:
Updated `get_usps_proofing_results_job_reprocess_delay_minutes` to `type: :integer`.